### PR TITLE
chore: remove OpenSearch warnings

### PR DESCRIPTION
**Pull request by grlimacan**
## Description

Now that OpenSearch is fully supported in Optimize with 8.7 / 3.15, we need to remove the warnings for the latest release. Note that the warnings still apply for 8.6 / 3.14 and below, therefore mostly only updating the most current docs

related to https://github.com/camunda/camunda/issues/23615
https://github.com/camunda/camunda/issues/23766

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: optimize/self-managed/optimize-deployment/advanced-features/import-guide.md [hunk 1]
```diff
@@ -191,15 +191,19 @@ Therefore, the Optimize `ProcessInstance` is an aggregation of the engine's hist
 :::note
 Optimize uses [nested documents](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html), the above mentioned data is an example of documents that are nested within Optimize's `ProcessInstance` index.
 
-Elasticsearch applies restrictions regarding how many objects can be nested within one document. If your data includes too many nested documents, you may experience import failures. To avoid this, you can temporarily increase the nested object limit in Optimize's [index configuration](./../configuration/system-configuration.md#index-settings). Note that this might cause memory errors.
+Elasticsearch and OpenSearch apply restrictions regarding how many objects can be nested within one document. If your data includes too many nested documents, you may experience import failures. To avoid this, you can temporarily increase the nested object limit in Optimize's [index configuration](./../configuration/system-configuration.md#index-settings). Note that this might cause memory errors.
```
Comment (PHWaechtler) [inline optimize/self-managed/optimize-deployment/advanced-features/import-guide.md:194]: There are still a bunch of other `Elasticsearch` occurrences in this file, shall we adjust those too?

File: optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-7.md [hunk 2]
```diff
@@ -64,8 +64,8 @@ REST API endpoint locations, timeouts, etc.
...
+| import.elasticsearchJobExecutorThreadCount\*              | 1             | Number of threads being used to process the import jobs per data type that are writing data to elasticsearch.                                                                                                                                                                                                                                                                                                                                                                                                             |
+| import.elasticsearchJobExecutorQueueSize\*                | 5             | Adjust the queue size of the import jobs per data type that store data to elasticsearch. If the value is too large it might cause memory problems.                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (PHWaechtler) [inline optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-7.md:68]: Lets also adjut the description of these (and any other shared DB settings) to say `database` or `elaticsearch/opensearch`

File: optimize_versioned_docs/version-3.14.0/self-managed/optimize-deployment/advanced-features/import-guide.md [hunk 3]
```diff
@@ -191,15 +191,19 @@ Therefore, the Optimize `ProcessInstance` is an aggregation of the engine's hist
 :::note
 Optimize uses [nested documents](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html), the above mentioned data is an example of documents that are nested within Optimize's `ProcessInstance` index.
 
-Elasticsearch applies restrictions regarding how many objects can be nested within one document. If your data includes too many nested documents, you may experience import failures. To avoid this, you can temporarily increase the nested object limit in Optimize's [index configuration](./../configuration/system-configuration.md#index-settings). Note that this might cause memory errors.
+Elasticsearch and OpenSearch apply restrictions regarding how many objects can be nested within one document. If your data includes too many nested documents, you may experience import failures. To avoid this, you can temporarily increase the nested object limit in Optimize's [index configuration](./../configuration/system-configuration.md#index-settings). Note that this might cause memory errors.
```
Comment (PHWaechtler) [inline optimize_versioned_docs/version-3.14.0/self-managed/optimize-deployment/advanced-features/import-guide.md:194]: Same here there are still more `Elasticsearch` mentions that we could rephrase to be more generic

**Review by PHWaechtler — CHANGES_REQUESTED** [review]
Warning removal lgtm, just got some small suggestions and leftover `Elasticsearch` mentions, though let me know if we're deliberately not changing all of them, unsure what the current state here is.

File: optimize/self-managed/optimize-deployment/advanced-features/import-guide.md (hunk 1 above)
Comment (grlimacan) [reply optimize/self-managed/optimize-deployment/advanced-features/import-guide.md:194]: Yep, well spotted. Done

File: optimize_versioned_docs/version-3.14.0/self-managed/optimize-deployment/advanced-features/import-guide.md [hunk 4]
```diff
@@ -131,11 +131,11 @@ During execution, the following steps are performed:
...
-   1. Poll a new page
-   2. Map entities and add an import job
-3. [Execute the import](#execute-the-import).
-   1. Poll a job
-   2. Persist the new entities to Elasticsearch
+3. Poll a new page
+4. Map entities and add an import job
+5. [Execute the import](#execute-the-import).
+6. Poll a job
+7. Persist the new entities to the database
```
Comment (PHWaechtler) [inline optimize_versioned_docs/version-3.14.0/self-managed/optimize-deployment/advanced-features/import-guide.md:138]: I think this indentation was changed accidentally

**Review by PHWaechtler — DISMISSED** [review]
Lgtm just that one indentation change needs undone 👍  👍

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**